pkg/config/setup/system_probe.go

```diff
@@ -438,6 +438,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault(join(gpuNS, "initial_process_sync"), true)
 	cfg.BindEnvAndSetDefault(join(gpuNS, "configure_cgroup_perms"), false)
 	cfg.BindEnvAndSetDefault(join(gpuNS, "enable_fatbin_parsing"), false)
+	cfg.BindEnvAndSetDefault(join(gpuNS, "ring_buffer_size_pages_per_device"), 64) // 32 pages = 128KB by default per device
```

Comment:
maybe better `ring_buffer_pages_per_gpu_device`?
also I think the comment doesn't match the default value